### PR TITLE
#1812 Added my name to credits

### LINKED ISSUE #1812: Yipeng(Andrew) Wang - Credits
### Description

Giving myself credit for the 
Astonishing
Incredible
Marvelous
Wonderful
Remarkable
Stupendous
Fantastic
Fabulous
Impressive
Phenomenal 
Breathtaking
Astounding
Extraordinary
Spectacular
Magnificent
Stunning
Exquisite
Miraculous
Wondrous
Dazzling
developing work I will be doing at NER!

### Acceptance Criteria

See my name in my favorite color on the credits page.

### Proposed Solution

Add my name and favorite color to the "names" array in the CreditsPage.tsx file

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by YPAndrew0907**
## Changes

I've added my name to the end of the list.

## Screenshots

image

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] No `yarn.lock` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

closes #1812

### CODE REVIEW

**Review by ryanhowe28 — CHANGES_REQUESTED** [review]
Can you put your name at the bottom. Also fill out the pull request form with your changes and screenshots of your change on the page.

**Review by YPAndrew0907 — COMMENTED** [review]
Name added at the last 

Image 2024-1-24 at 20 39

File: src/frontend/src/pages/CreditsPage/CreditsPage.tsx [hunk 1]
```diff
@@ -207,6 +207,14 @@ const CreditsPage: React.FC = () => {
     { name: 'Vinay Pillai', color: '#42458e' },
     { name: 'Jay Jung', color: '##344739' },
     { name: 'Will Anderson', color: '#ebb145' },
+    {
+      name: ' Yipeng (Andrew) Wang',
```
Comment (jackturner106) [inline src/frontend/src/pages/CreditsPage/CreditsPage.tsx:211]: Could you remove the space before your name?

**Review by jackturner106 — COMMENTED** [review]
This looks great, I really like the gradient! I did leave one small change, other than that it looks good.

**Review by RChandler234 — APPROVED** [review]
LGTM 🔥

**Pull request merged**